cli/manifest/store/store.go

```diff
@@ -152,28 +151,3 @@ func makeFilesafeName(ref string) string {
 	fileName := strings.Replace(ref, ":", "-", -1)
 	return strings.Replace(fileName, "/", "_", -1)
 }
-
-type notFoundError struct {
-	object string
-}
-
-func newNotFoundError(ref string) *notFoundError {
-	return &notFoundError{object: ref}
-}
-
-func (n *notFoundError) Error() string {
-	return fmt.Sprintf("No such manifest: %s", n.object)
-}
-
-// NotFound interface
-func (n *notFoundError) NotFound() {}
```

Comment:
Wondering; instead of a new error-type, could we change the errors to use these, and use `errdefs.IsNotFound()` (etc)? https://github.com/docker/cli/blob/c1733165159c08101adb0e1f120c7181533550ef/vendor/github.com/docker/docker/errdefs/is.go#L31-L35